src/utils/beneficial.owner.individual.ts

```diff
@@ -184,21 +185,28 @@ export const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDat
   data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
   data[CeasedDateKey] = reqBody["is_still_bo"] === '0' ? mapFieldsToDataObject(reqBody, CeasedDateKeys, InputDateKeys) : {};
 
+  setBoNocData(data);
+
+  data[IsOnSanctionsListKey] = (data[IsOnSanctionsListKey]) ? +data[IsOnSanctionsListKey] : '';
+
+  data[ID] = id;
+
+  return data;
+};
+
+const setBoNocData = (data: ApplicationDataType) => {
   // It needs concatenations because if in the check boxes we select only one option
   // nunjucks returns just a string and with concat we will return an array.
   data[BeneficialOwnerNoc] = (data[BeneficialOwnerNoc]) ? [].concat(data[BeneficialOwnerNoc]) : [];
```

Comment:
Is it possible to change the template to return an array?

Reply:
Possibly, but I don't think we should do that as part of this change. We're simply following the same approach as we've done elsewhere.